.devcontainer/build.sh

```diff
@@ -17,8 +17,8 @@
 #
 
 HERE="$(dirname "$0")"
-CHIP_ROOT="$(realpath "$HERE"/..)"
-BUILD_VERSION="latest"
+CHIP_ROOT="$(git rev-parse --show-toplevel)"
```

Comment:
I'm not sure whether we can assume that repo was cloned with git. What if someone has downloaded a zip from github?